packages/wonder-blocks-tooltip/src/components/tooltip-bubble.tsx

```diff
@@ -84,6 +84,7 @@ export default class TooltipBubble extends React.Component<Props, State> {
             >
                 <View style={styles.content}>{children}</View>
                 <TooltipTail
+                    show={true}
```

Comment:
Seems like we could remove this, since it default to true?

Reply:
Yes, totally. I made this change before I set the default. Good catch